### PR TITLE
Updated the CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ejmoore99**
Added content to the "Your First Code Contribution" & "Improving The Documentation" in the CONTRIBUTING.md file with a guide on how to contribute for the first time.

Screenshot (92)
Screenshot (93)

### CODE REVIEW

**Comment by lucavallin** [comment]
Hey! Thanks for the PR. It seems there are a few "typos" like "bashCopy... git clone...". If you fix those, we can merge it.

**Comment by ejmoore99** [comment]
Will get right on it!

**Comment by ejmoore99** [comment]
Sent a new PR with the bash typos fixed!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -147,11 +147,93 @@ Enhancement suggestions are tracked as [GitHub issues](https://github.com/lucava
...
+**Step 1 Fork the Repository:**
+
+- Go to the GitHub page of the project you want to contribute to.
+- Click on the "Fork" button at the top right corner to create a copy of the repository under your GitHub account.
+
+**Step 2 Clone the Repository:**
+
+- Clone the forked repository to your local machine using the command:
+
+bash </br>
```
Comment (lucavallin) [inline CONTRIBUTING.md:161]:
```suggestion

```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -147,11 +147,93 @@ Enhancement suggestions are tracked as [GitHub issues](https://github.com/lucava
...
+git clone https://github.com/your-username/repo-name.git
```
Comment (lucavallin) [inline CONTRIBUTING.md:162]:
```suggestion
```sh
git clone https://github.com/your-username/repo-name.git
```
```

**Comment by lucavallin** [comment]
Hey @ejmoore99, thanks! You probably want to rewrite those commands like this:

```sh
git clone ...
```

Read more on Markdown code blocks [here](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks).

**Comment by ejmoore99** [comment]
> Hey @ejmoore99, thanks! You probably want to rewrite those commands like this:
> 
> ```shell
> git clone ...
> ```
> 
> Read more on Markdown code blocks [here](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks).

I have updated it to display in code blocks. New PR sent out. Thank you!

**Comment by lucavallin** [comment]
Closing for https://github.com/lucavallin/verto/pull/442.

**Pull request closed without merging**